### DISCUSSION TITLE
PDO Extension not loaded

### DISCUSSION THREAD

**Discussion by ZephyrOff** [Kovah/LinkAce#592] [Support] [answered]
Hello,

we are installing LinkAce on one of our servers (php8.2, mariadb 10.9.4) and we are having a problem with the dependency check. All extensions are taken into account, except php-pdo. I checked by doing a php -m but the module is loaded. Has anyone encountered this problem before or have an idea?
![image](https://user-images.githubusercontent.com/55457378/209976067-e1523dbc-9b0e-4da4-9682-76834a205cb9.png)

**Comment by martadinata666** [comment, marked as answer]
I think it need both PDO and PDO mysql, 
my `php -m` show both this modules. 
```
PDO
pdo_mysql
```

**Reply by ZephyrOff** [reply]
It's good now. Thanks
